debian/jessie/foreman-installer/changelog

```diff
@@ -1,3 +1,39 @@
+foreman-installer (1.14.2-1) stable; urgency=low
+
+  * 1.14.2 released
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Thu, 30 Mar 2017 10:15:41 +0200
```

Comment:
Please re-check against deb/1.14, I'm pretty sure these have different dates and authors!